applications/nrf5340_audio/src/bluetooth/ble_acl_common.c

```diff
@@ -213,3 +230,163 @@ int ble_acl_common_init(void)
 
 	return 0;
 }
+
+static enum ble_hci_vs_tx_power selection_to_enum_val(int selection)
+{
+	switch (selection) {
+	case 0:
+		return BLE_HCI_VSC_TX_PWR_0dBm;
+	case 1:
+		return BLE_HCI_VSC_TX_PWR_Neg1dBm;
+	case 2:
+		return BLE_HCI_VSC_TX_PWR_Neg2dBm;
+	case 3:
+		return BLE_HCI_VSC_TX_PWR_Neg3dBm;
+	case 4:
+		return BLE_HCI_VSC_TX_PWR_Neg4dBm;
+	case 5:
+		return BLE_HCI_VSC_TX_PWR_Neg5dBm;
+	case 6:
+		return BLE_HCI_VSC_TX_PWR_Neg6dBm;
+	case 7:
+		return BLE_HCI_VSC_TX_PWR_Neg7dBm;
+	case 8:
+		return BLE_HCI_VSC_TX_PWR_Neg8dBm;
+	case 9:
+		return BLE_HCI_VSC_TX_PWR_Neg12dBm;
+	case 10:
+		return BLE_HCI_VSC_TX_PWR_Neg16dBm;
+	case 11:
+		return BLE_HCI_VSC_TX_PWR_Neg20dBm;
+	case 12:
+		return BLE_HCI_VSC_TX_PWR_Neg40dBm;
+	default:
+		return BLE_HCI_VSC_PRI_EXT_ADV_MAX_TX_PWR_DISABLE;
```

Comment:
We should not include this setting, since this is dedicated for ble_hci_vsc_set_pri_ext_adv_max_tx_pwr

Reply:
I'm using it as an error value, but I guess I can rather add a new value to the enum instead